src/locales/en/en.json

```diff
@@ -535,6 +535,14 @@
       }
     ]
   },
+  "aboutssiagent": {
+    "done": "Done",
+    "intro": {
+      "title": "About SSI agent",
+      "text": ""
```

Comment:
About SSI agent modal is using dummy content on figma now. I asked Robert and He told me for now we don't have this information.  Therefore, I create empty key here and update it after Robert got content for this modal